netlify.toml

```diff
@@ -237,3 +237,11 @@
   from = "/pub"
   to = "/pubcrawl"
 
+# Roller Skating 2023-11-16
+[[redirects]]
+  from = "/rollerskating"
+  to = "https://www.skiddle.com/whats-on/Birmingham/Roller-Jam/Get-Down---Free-Entry-and-241-Cocktails-6pm-2am-For-Ages-18/36707712/"
+
+[[redirects]]
+  from = "/skate"
+  to = "https://www.skiddle.com/whats-on/Birmingham/Roller-Jam/Get-Down---Free-Entry-and-241-Cocktails-6pm-2am-For-Ages-18/36707712/"
```

Comment:
This does not need to go to the original link. It should follow the other redirects where it goes to the single source of truth redirect (in this case `"/rollerskating"`

Reply:
Ngl, I think that's minor enough to not matter, but made the change anyway.

The voting links should be changed too in that case though.